docs-examples/example-java/build.gradle

```
@@ -2,3 +2,7 @@ plugins {
     id 'io.micronaut.build.internal.nats-native-tests'
     id("io.micronaut.build.internal.java-base")
 }
+
+dependencies {
+    implementation(projects.testSuiteNatsUtils)
```

why in the main classpath?

```suggestion
    testImplementation(projects.testSuiteNatsUtils)
```

---

https://github.com/micronaut-projects/micronaut-nats/pull/652/commits/bfd5e2bbd6358c2c83ee018f7178033e1819cb3c